components/connector/Dockerfile

```diff
@@ -13,7 +13,7 @@ RUN mkdir /app && mv ./main /app/main && mv ./licenses /app/licenses
 
 
 
-FROM alpine:3.15.0
+FROM alpine:3.15.3
```

Comment:
For some reason I can't comment on L1, but bump the golang version here as well

Reply:
It is not trivial and require refactoring. We agree with @NickyMateev to do this in separate task.